pkgs/by-name/go/gokey/package.nix

```
@@ -1,16 +1,18 @@
 { lib, buildGoModule, fetchFromGitHub }:
-buildGoModule {
+buildGoModule rec {
   pname = "gokey";
-  version = "0.1.2-unstable-2023-11-16";
+  version = "0.1.3";
+
+  patches = [ ./version.patch ];
 
   src = fetchFromGitHub {
     owner = "cloudflare";
     repo = "gokey";
-    rev = "26fcef24d123e0eaf7b92224e6880f529f94aa9f";
-    hash = "sha256-nt4fO8NKYfRkpoC1z8zDrEZC7+fo6sU/ZOHCMHIAT58=";
+    rev = "refs/tags/${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

My bad, the tag comes with a v prefix.

---

Hmm... weird, it did build for me even without the `v`, but it's added now.

---

That's a footgun with FOD (fixed output derivations), as long as the hash doesn't change, it never get rebuilt.